tests/BenchmarkDotNet.IntegrationTests.ManualRunning/BenchmarkDotNet.IntegrationTests.ManualRunning.csproj

```diff
@@ -0,0 +1,35 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="..\..\build\common.props" />
+  <PropertyGroup>
+    <AssemblyTitle>BenchmarkDotNet.IntegrationTests.ManualRunning</AssemblyTitle>
+    <TargetFrameworks>net46;netcoreapp2.1</TargetFrameworks>
```

Comment:
Does it make sense to add `netcoreapp2.0` here?

Reply:
As of today not, because all the tests that we have today in this project are generating a .NET Core 2.1 apps. In the future, when we add more tests to it, it's going to make sense.